app/controllers/application_controller.rb

```diff
@@ -16,6 +16,35 @@ def current_user
   end
 
   def authenticate
-    redirect_to '/signin' unless current_user
+    if current_user
+      set_connection
+    else
+      redirect_to '/signin'
+    end
+  end
+
+  def current_user_info
+    current_user['info']
+  end
+
+private
+
+  def set_connection
```

Comment:
`def add_token_to_connections`?

Reply:
👍 to @tvararu's suggestion

Reply:
going with `add_token_to_connection` as pluralised has no meaning